ruma-events/src/lib.rs

```diff
@@ -282,6 +286,12 @@ pub struct Unsigned {
     /// is the same one which sent it.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub transaction_id: Option<String>,
+
+    /// Server-compiled information from other events relating to this event.
+    #[cfg(feature = "unstable-pre-spec")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "m.relations")]
```

Comment:
The two serde attributes can be combined into one: `#[serde(rename = "...", skip_serializing_if = "...")]`